src/dojos/cd-find-dojo.vue

```diff
@@ -80,6 +95,9 @@
       allActiveDojos() {
         return this.allDojos.filter(dojo => dojo.stage !== 4);
       },
+      queryString() {
+        return this.$route.query.q ? `‘${this.$route.query.q}’` : this.$t('you');
```

Comment:
Won't that trigger a search with q = "you" ?

Reply:
No this is the string that is returned when you click search or detect my location. When there is no query it returns you, otherwise it returns the query contained in quote marks.

Reply:
Apologies, you're right. I read that as an assignement.